src/olympia/reviewers/views.py

```diff
@@ -439,6 +439,19 @@ def determine_channel(channel_as_text):
     return channel, content_review
 
 
+def other_queue_has_unreviewed_versions(addon, channel):
+    """Return True if the addon has unreviewed versions in another queue."""
+    versions = addon.versions.filter(
+        channel=amo.CHANNEL_LISTED
+        if channel == amo.CHANNEL_UNLISTED
+        else amo.CHANNEL_UNLISTED,
+    )
```

Comment:
I guess you can use `exclude` instead
```suggestion
    versions = addon.versions.exclude(channel=channel)
```